services/characterlcdscreen.md

```diff
@@ -0,0 +1,49 @@
+# Character LCD Screen
```

Comment:
alphanumeric display? or character display? 

I would drop LCD, since it may as well be LED display. I could also imagine implementing this on a 5x5 microbit style display

Reply:
I was thinking about those specifically: https://learn.adafruit.com/character-lcds/overview

Reply:
I think the 5x5 LED screen is something else

Reply:
ok, ignoring the 5x5 scrolling display, there are character displays that are LEDs (eg on buses; i'm sure there are other examples); also even if it's a graphical display, it could still expose the character service

Reply:
ok, i'll add back backlight once we have a hSV type.